components/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import Headline from './Headline'
+
+import { 
+    FOOT_ITEMS,
+    GITHUB_URL,
+    LINKEDIN_URL,
+    SETLIFE_NETWORK_URL,
+    TWITTER_URL,
+    YOUTUBE_URL,
+} from '../constants'
+import {
+    SETLIFE,
+    SOLUTIONS,
+    SETLIFE_SOLUTIONS_COPYRIGHT,
+    SETLIFE_NETWORK,
+} from '../constants/strings'
+
+const renderFootItems = () => {
+    return (
+        FOOT_ITEMS.map((n, i) => {
+            const logoName = n.substring(50).replace('.png', '')
+            const URL = () => {
+                if (logoName == 'Youtube') {
+                    return YOUTUBE_URL
+                } else if (logoName == 'Twitter') {
+                    return TWITTER_URL
+                } else if (logoName == 'Github') {
+                    return GITHUB_URL
+                } else if (logoName == 'Linkedin') {
+                    return LINKEDIN_URL
+                }
+            }
```

Comment:
Why not adding `URL` into `FOOT_ITEMS` directly?

Reply:
I tried to do it and do a map to assign it but the link I was getting redirected to something like this `localhost:3000/..https://GithubLink.com..` Instead of getting redirected directly to the page

Reply:
**Update**
Manage to make it work using just 1 Array